.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
   - "8.9.4"
-script:
-  - npm install
+install:
```

Comment:
Are these changes required as part of this ticket?

Reply:
I had issues with travis build terminating without running tests.
I researched and the solution was to restructure the travis.yml file.

Reply:
That requires a separate ticket and pull request. Do you still remember the article i shared with you over the weekend? 😊